docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+      "execution_count": 2,
+      "metadata": {
+        "id": "hpD0FvNBXVbM"
+      },
+      "outputs": [],
+      "source": [
+        "import numpy as np\n",
+        "import polars as pl\n",
+        "import opendp.prelude as dp\n",
+        "import matplotlib.pyplot as plt\n",
+        "import seaborn as sns\n",
+        "\n",
+        "dp.enable_features(\"contrib\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2aBAZ5AmXVbM"
+      },
+      "source": [
+        "If the \"sample_FR_LFS.csv\" file exists, then follow the code for importing the dataset as is. Otherwise follow the instructions from Preprocessing.ipynb to compile the dataset."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "id": "wjAtK28VXVbM"
+      },
+      "outputs": [],
+      "source": [
+        "# reading in the data\n",
+        "df = pl.scan_csv(\"sample_FR_LFS.csv\", infer_schema_length=1000, ignore_errors=True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f6PEZWpzofFn"
+      },
+      "source": [
+        "For more information on defining the compositor, see the Part 1 Notebook on Data Exploration methods!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "id": "3ltU268CXVbM"
+      },
+      "outputs": [],
+      "source": [
+        "context = dp.Context.compositor(\n",
+        "    data=df,\n",
+        "    privacy_unit=dp.unit_of(contributions=1),\n",
+        "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+        "    split_evenly_over=10,\n",
+        "    margins={\n",
+        "        (\"AGE\", ): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+        "        (): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000),\n",
+        "    },\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XxM09cKUXVbN"
+      },
+      "source": [
+        "## Using the Quantile Method Directly"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Before we get into the quantile method, let's try out the median method directly. We also will need to impute the null values with the mean. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 18,
+      "metadata": {
+        "id": "eryzri-4XVbN"
+      },
+      "outputs": [],
+      "source": [
+        "mean_age = df.select(pl.col(\"AGE\")).mean().collect()\n",
+        "candidates = list(range(100))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Actual Median: 47.0\n",
+            "Differentially Private Median: 46\n"
```

Comment:
A little surprised that one is float and one is int? Maybe file an issue from this comment if you don't understand why they're different -- It doesn't really matter, but it's a little surprising.